### DISCUSSION TITLE
NOt able to start inbucket

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#362] [General]
hi i did all posibility to install inbucket on my ubuntu machinchi but it sho fail always.

Loaded: loaded (/etc/systemd/system/inbucket.service; enabled; vendor pre>

     Active: failed (Result: exit-code) since Fri 2023-05-05 09:48:12 IST; 8mi>

    Process: 716 ExecStartPre=/bin/mkdir -p /var/local/inbucket (code=exited, >

    Process: 722 ExecStartPre=/bin/chown daemon:daemon /var/local/inbucket (co>

    Process: 735 ExecStart=/usr/local/bin/inbucket (code=exited, status=203/EX>

   Main PID: 735 (code=exited, status=203/EXEC)

        CPU: 15ms

May 05 09:48:12 sachin-VirtualBox systemd[1]: Starting Inbucket Disposable Ema>

May 05 09:48:12 sachin-VirtualBox systemd[1]: Started Inbucket Disposable Emai>

May 05 09:48:12 sachin-VirtualBox systemd[735]: inbucket.service: Failed to lo>

May 05 09:48:12 sachin-VirtualBox systemd[735]: inbucket.service: Failed at st>

May 05 09:48:12 sachin-VirtualBox systemd[1]: inbucket.service: Main process e>

May 05 09:48:12 sachin-VirtualBox systemd[1]: inbucket.service: Failed with re

**Comment by jhillyerd** [comment]
When viewing those logs, you should be able to tap your right cursor key to see more of each line of error messages -- It's not clear what is failing.

**Reply by techsachin1** [reply]
× inbucket.service - Inbucket Disposable Email Service

     Loaded: loaded (/etc/systemd/system/inbucket.service; enabled; vendor preset: enabled)

     Active: failed (Result: exit-code) since Mon 2023-05-08 12:39:57 IST; 13min ago

    Process: 7873 ExecStartPre=/bin/mkdir -p /var/local/inbucket (code=exited, status=0/SUCCESS)

    Process: 7874 ExecStartPre=/bin/chown daemon:daemon /var/local/inbucket (code=exited, status=0/SUCCESS)

    Process: 7875 ExecStart=/usr/local/bin/inbucket (code=exited, status=203/EXEC)

   Main PID: 7875 (code=exited, status=203/EXEC)

        CPU: 11ms



May 08 12:39:57 sachin-VirtualBox systemd[1]: Starting Inbucket Disposable Email Service...

May 08 12:39:57 sachin-VirtualBox systemd[1]: Started Inbucket Disposable Email Service.

May 08 12:39:57 sachin-VirtualBox systemd[7875]: inbucket.service: Failed to locate executable /usr/local/bin/inbucket: Is a directory

May 08 12:39:57 sachin-VirtualBox systemd[7875]: inbucket.service: Failed at step EXEC spawning /usr/local/bin/inbucket: Is a directory

May 08 12:39:57 sachin-VirtualBox systemd[1]: inbucket.service: Main process exited, code=exited, status=203/EXEC

May 08 12:39:57 sachin-VirtualBox systemd[1]: inbucket.service: Failed with result 'exit-code'.

please help me to install and configure inbucket.. ihave to configure it for my organisation

**Comment by jhillyerd** [comment]
There are a number of different ways to install inbucket, so it's difficult for me to tell you exactly how to install it.  For Ubuntu, you'd be best off installing the debian package.  (Download the .deb [here](https://github.com/inbucket/inbucket/releases), and see the debian instructions [here](https://inbucket.org/installation/linux.html))

Alternatively, if you and your organization are already familiar with Docker, we provide prebuilt images here: https://hub.docker.com/r/inbucket/inbucket/

**Comment by techsachin1** [comment]
thanks for reply
kindly share how to configure it locally

**Comment by techsachin1** [comment]
Please how to configure to local machine. i have sepd whole 2 days but not
able to configure. kindly share pre-requisite for it also
